README.md

```diff
@@ -1,2 +1,97 @@
-# Ride-My-Way2
-A Web Application that enable users(CarTax Operators) to create an account and be able to create ride offers. Passengers will be able to view the ride offers and request to join. 
+[![Build Status](https://travis-ci.org/Mutuba/Ride-My-Way2.svg?branch=feature)](https://travis-ci.org/Mutuba/Ride-My-Way2)
+[![Coverage Status](https://coveralls.io/repos/github/Mutuba/Ride-My-Way2/badge.svg)](https://coveralls.io/github/Mutuba/Ride-My-Way2)
+
+## Ride-My-Way2
+
+A Web Application that enable users(CarTax Operators) to create an account and be able to create ride offers. Passengers will be able to view the ride offers and request to join:
+
+- Register an account and Login into it.
+- Register, Update and delete a Ride .
+- View all Rides.
+- View One Rides.
+- Post Requests for rides.
+- View all Requests for rides
+- Change users password
+
+## Prerequisites
+
+- Python 3.6 or a later version
+
+## Installation
+Clone the repo.
+```
+$ git clone https://github.com/mutuba/Ride-My-Way2.git
+```
+and cd into the folder:
+```
+$ /Ride-My-Way
+```
+## Virtual environment
+Create a virtual environment:
+```
+mkvirtualenv <virtual environment name>
```

Comment:
Consider giving an example. For instance `myenv`. This will be more helpful to non-technical who might want to use your app.

Reply:
Thanks